Files/ViewModels/ColumnsViewModel.cs

```diff
@@ -120,14 +119,15 @@ private void SetColumnSizeProportionally(double factor)
 
     public class ColumnViewModel : ObservableObject
     {
-
+        [JsonIgnore]
```

Comment:
This too should be moved on the public property (or if you know that it works regardless I'd learn something).